### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by hunter4466**
Developed the main structure of a to-do list with HTML, javascript, and CSS.
For this project, I used Webpack.

### CODE REVIEW

**Comment by mricanho** [comment]
Hi @hunter4466, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has the node_modules, remember that you need to add node_modules to the .gitignore file, double-check the [repo rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by mricanho** [comment]
Hi @hunter4466, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that still has the node_modules in the repo. This happens because you committed the directory before adding it to the .gitignore files. [Check this link](https://stackoverflow.com/questions/45400361/why-is-gitignore-not-ignoring-my-files) and follow the instructions to remove node_modules from the repository.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
-# todolist_project
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To do list - project
+
+> Creating an Awesome Books Page
+
+![screenshot](./Assets/Images/Screenshots/preview1.png)
```
Comment (mricanho) [inline README.md]: - [ ] [OPTIONAL] This link is broken, kindly fix it

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+- Html, Javascript, CSS
+- Webpack
+- Developed in VSCode
+
+
+To get a local copy up and running follow these simple example steps.
+Run:
+- `git clone https://github.com/hunter4466/todolist_project.git`
+- `cd todolist_project`
+- Open ./dist/index.html in browser
```
Comment (mricanho) [inline README.md]: - [ ] You need to add steps to run your project, there's no mention about npm install or npm start

File: package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "todolist_project",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```
Comment (mricanho) [inline package.json]: - [ ] You need to add more scripts, remember that the basics are for development and production, the build script could be for development but you are missing the one for production

File: src/Assets/Javascript/linkedList.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    let currentNode = this.head;
+    allArray.push(currentNode.value);
+    while (currentNode.nextNode) {
+      currentNode = currentNode.nextNode;
+      allArray.push(currentNode.value);
+    }
+
+    return allArray;
+  }
+
```
Comment (mricanho) [inline src/Assets/Javascript/linkedList.js:107]: - [ ] Add a function to populate the HTML list with a couple of example tasks (I know that your todo list works right now, good work btw)

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @hunter4466, 

Good job so far! Your todo list it's working already and that's excellent news, node_modules it's gone for good and the style of your app it's super similar to the target one  🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

doing good

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hemantso — APPROVED** [review]
Hi @hunter4466 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- There is an issue in the terminal when running the command `npm start`. It is due to the 'mode' option has not been set. Set the 'mode' option to 'development' or 'production' to enable defaults for each environment. Please try to fix this issue. [Reference](https://webpack.js.org/configuration/mode/)


Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**